### PR TITLE
release/1.5

### LINKED ISSUE #12680: Release 1.5
### Prepare the project

Do some housekeeping on GitHub in the [main repo](https://github.com/OrchardCMS/OrchardCore).

- [ ] Close remaining issues for the version (including merging corresponding pull requests if suitable) or assign them to the next one.
- [ ] Assign all issues that were closed for an upcoming version (including a wildcard version like "1.0.x") to this version (milestone).

### Prepare the code

Update the source so everything looks like on the new version.

- [x] Create a `release/<version name>` branch out of `main`.
- [x] Update code generation templates in `OrchardCore.ProjectTemplates`.
- [x] Update the `OrchardCore.Commons.props` file: Update `VersionPrefix` for release versions (like "1.0.0") and `VersionSuffix` for pre-release versions (like "rc2", for the full version to be e.g. "1.0.0-rc2").
- [x] Update module versions in `ManifestConstants`.

### Prepare and publish Orchard Core Translations

Update everything in the [Translations project](https://github.com/OrchardCMS/OrchardCore.Translations). Only do this once all the code changes are done since localized strings can change until then.

- [x] Update .po files with [PoExtractor](https://github.com/lukaskabrt/PoExtractor). This will also update [Crowdin](https://crowdin.com/project/orchard-core).
- [x] Publish the new version on NuGet.
- [x] Update the `OrchardCore.Translations.All` package reference in the main repo's _src/OrchardCore.Build/Dependencies.props_ file to refer to the new NuGet package.

### Test the release

Make sure everything works all right.

- [x] Make sure that [OrchardCore.Samples works](https://github.com/OrchardCMS/OrchardCore.Samples).
- [x] - [Creating a modular ASP.NET Core application](https://docs.orchardcore.net/en/latest/docs/guides/create-modular-application-mvc/)
    - [Creating an Orchard Core CMS website](https://docs.orchardcore.net/en/latest/docs/guides/create-cms-application/)
    - [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/docs/guides/decoupled-cms/)

### Prepare the documentation

Update the docs so they contain information about the new release so once the release is out you'll just need to point to new information.

- [x] Create release notes in a specific documentation section. You can take the previous release notes as a template.
    - Overview of the release's highlights and goals. What do you want people to remember this release for?
    - Prerequisites. What framework version do you need, anything else to work with Orchard?
    - Upgrade steps, any migration necessary from previous versions, breaking changes.

### Publish the release

Do the harder parts of making the release public. This should come after everything above is done.

- [ ] Merge `release/<version name>` to `master`.
    - Merges to `master` need two approvals so you'll need to create a pull request.
    - Merge it as a merge commit, not squash merge.
- [ ] Tag `main` with the full version name, including the prefix and suffix (e.g. "1.0.0-rc2").
- [ ] Test the [guides](https://docs.orchardcore.net/en/latest/docs/guides/) with the packages now automatically published to NuGet. Test at least the following guides:
    - [Creating a modular ASP.NET Core application](https://docs.orchardcore.net/en/latest/docs/guides/create-modular-application-mvc/)
    - [Creating an Orchard Core CMS website](https://docs.orchardcore.net/en/latest/docs/guides/create-cms-application/)
    - [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/docs/guides/decoupled-cms/)
- [ ] Update [Try Orchard Core](https://github.com/OrchardCMS/TryOrchardCore).

### Publicize the release

Let the whole world know about our shiny new release. Savor this part! These steps will make the release public so only do them once everything else is ready.

- [ ] Update the documentation to mention the version in all places where the latest version is referenced, for example, but not limited to (do a search for the package version string): [Status in the root README](https://docs.orchardcore.net/en/latest/#status), CLI templates, commands, the [Creating a new decoupled CMS Website](https://docs.orchardcore.net/en/latest/docs/guides/decoupled-cms/) guide.
- [ ] Update the tagged [release](https://github.com/OrchardCMS/OrchardCore/releases) on GitHub: Change its title to something more descriptive (e.g. "Orchard Core 1.0.0 RC 2"), add a link in its description to the release notes in the documentation (something like `For details on this version see the [release notes in the documentation](link here).`). Add a link to this release under [Status in the root README](https://docs.orchardcore.net/en/latest/#status).
- [ ] Publish a blog post on the website.
- [ ] Ask to publish a blog post on [DevBlogs](https://devblogs.microsoft.com/).
- [ ] Ask to publish a blog post on [.NET Foundation News](https://dotnetfoundation.org/news).
- [ ] Tweet

### PR DESCRIPTION
**Pull request by sebastienros**
Closes #12680

### CODE REVIEW

File: src/OrchardCore.Build/Dependencies.props [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <PackageManagement Include="NodaTime" Version="3.1.4" />
     <PackageManagement Include="OpenIddict.AspNetCore" Version="3.1.1" />
     <PackageManagement Include="OpenIddict.Core" Version="3.1.1" />
-    <PackageManagement Include="OrchardCore.Translations.All" Version="1.4.0" />
+    <PackageManagement Include="OrchardCore.Translations.All" Version="1.0.1-beta-20024" />
```
Comment (hishamco) [inline src/OrchardCore.Build/Dependencies.props:50]: Why it's `1.0.1-beta`?!!

Comment (sebastienros) [reply src/OrchardCore.Build/Dependencies.props:50]: crowdin only, just incrementing the build number

**Comment by Skrypt** [comment]
@rprouse @sebastienros Any idea why the DNF license cannot be found on the OC project?

**Comment by rprouse** [comment]
@Skrypt The .NET Foundation uses either an assignment model or a contribution model for on-boarding new projects. Under the assignment model, a project transfers ownership of the copyright to the .NET Foundation. Under the contribution model, a project retains ownership of the copyright, but grants the .NET Foundation a broad license to the project's code and other intellectual property.

I would need to check, but my guess is that OC is under the contribution model.

**Comment by ns8482e** [comment]
> Does the release align with .NET 7.0 or not?

1.5 is still on .NET 6

**Comment by hishamco** [comment]
> 1.5 is still on .NET 6

Yep, I thought Seb waiting for upcoming major release

**Comment by Skrypt** [comment]
There is a PR for .NET 7.0 and it is not planned for this milestone. Though, we need to understand what is happening with the licence/cla from DNF. Since last week every new PR has had issues with recognizing the license/cla agreement that we as contributors to the project have signed at some point in time. So, while I'm not an admin of this repository; I think we will need to wait on @sebastienros. I also added @rprouse to the discussion so that we be able to at least get a lead on what the issue might possibly be. I'm sorry in advance to bother you both about this, I just have no idea what is happening exactly. So far, @ns8482e  tested to know if there was a licence associated with this repository and we found none by testing with this url: https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=12671

**Pull request merged**